crates/testing/src/test_builder.rs

```diff
@@ -152,14 +152,14 @@ impl TestMetadata {
             completion_task_description: CompletionTaskDescription::TimeBasedCompletionTaskBuilder(
                 TimeBasedCompletionTaskDescription {
                     // Increase the duration to get the expected number of successful views.
-                    duration: Duration::new(340, 0),
+                    duration: Duration::new(3400, 0),
                 },
             ),
             overall_safety_properties: OverallSafetyPropertiesDescription {
                 ..Default::default()
             },
             timing_data: TimingData {
-                next_view_timeout: 5000,
+                next_view_timeout: 50000,
```

Comment:
Same deal here, it's probably fine to keep as is.  If libp2p is specifically slow then we can change the timeout just for those tests, and move them out of macro tests.  E.g. adding the tests to `crates/testing/tests/libp2p.rs` and overriding the default `TimingData` for those tests